Applications/Spire/Source/Ui/RegionBox.cpp

```diff
@@ -125,6 +173,19 @@ void RegionBox::resizeEvent(QResizeEvent* event) {
   QWidget::resizeEvent(event);
 }
 
+void RegionBox::region_to_list_of_tags(const Region& region,
```

Comment:
Why is this a method?

Reply:
Because m_query_model is used inside the function. The function could become a free function by passing the query model in it.